src/TestUtils/src/UITest.Appium/Actions/AppiumLifecycleActions.cs

```diff
@@ -61,12 +61,6 @@ CommandResponse LaunchApp(IDictionary<string, object> parameters)
 					{ "bundleId", _app.GetAppId() },
 				});
 			}
-			else if (_app.GetTestDevice() == TestDevice.Windows)
-			{
-#pragma warning disable CS0618 // Type or member is obsolete
-				_app.Driver.LaunchApp();
-#pragma warning restore CS0618 // Type or member is obsolete
-			}
 			else 
 			{
 				_app.Driver.ActivateApp(_app.GetAppId());
```

Comment:
Does the new code path now work with packaged and unpakcaged apps?

Reply:
Take a look to https://github.com/dotnet/maui/pull/22117#discussion_r1592055635